### DISCUSSION TITLE
The question of using deepks

### DISCUSSION THREAD

**Discussion by XiankeYue** [deepmodeling/deepks-kit#50] [Q&A]
Hello, can deepks combined with pyscf calculate the charged system, can deepks model be used for structural optimization calculation, what parameters need to be modified or called, thanks.

**Comment by ouqi0711** [comment]
In principle DeePKS (both with PySCF for molecular system and for with ABACUS for periodic system) supports charged systems. If you train the DeePKS model with force label, then the DeePKS model can provide force calculation with the same accuracy as your target method, and can thus be used for structural optimization. To do that, you just need to train the model with force label enabled.
